conf/ocsci/production-vsphere-upi-ocs4qe-lab.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
If these are not true production values should we remove `production-` from the filename?

Reply:
Done